examples/pageon.js

```diff
@@ -0,0 +1,38 @@
+import { browser } from 'k6/x/browser';
+import { check } from 'k6';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+            type: 'chromium',
+        },
+      },
+    },
+  },
+  thresholds: {
+    checks: ["rate==1.0"]
+  }
+}
+
+export default async function() {
+  const page = browser.newPage();
+  
+  try {
+    await page.goto('https://test.k6.io/');
+
+    page.on('console', msg => {
+        check(msg, {
+            'assertConsoleMessageType': msg => msg.type() == 'log',
+            'assertConsoleMessageText': msg => msg.text() == 'this is a console.log message 42',
+            'assertConsoleMessageArgs': msg => msg.args()[0].jsonValue() == 'this is a console.log message',
```

Comment:
Is `msg.args()[1]` equal to `42`?

Reply:
Yes, should we add it to the example?

Reply:
Added it in [de28297](https://github.com/grafana/xk6-browser/pull/1006/commits/de282976e05a05de8453fef13d433d0da92a84e6).